contrib/test/integration/README.md

```diff
@@ -14,7 +14,7 @@ When running `main.yml`, three tags are present:
 
 The playbooks assume the following things about your system:
 
- - on RHEL, the server and extras repos are configured and certs are present
+ - on RHEL, the EPEL, server, and extras repos are configured and certs are present.
```

Comment:
While I prefer periods terminating list entries in general, the remainder of this list does not use terminal punctuation.  I'd rather stay consistent and leave the trailling period off, or move the whole list over to trailing periods.

Reply:
And it should begin with a capital letter, and I'm sure plenty of other issues as well :grinning: 

okay, I'll give it some more polish, thanks for the feedback.